src/main/groovy/io/cify/framework/recording/RecordingController.groovy

```diff
@@ -87,13 +90,37 @@ class RecordingController {
         }
     }
 
+    /**
+     * Stop flick video recording
+     *
+     * @param device - device to stop flick recording
+     * @param farmUrl - device farm url
+     * @param sessionId - device session id
+     * */
+    public static void stopFlickRecording(Device device, String farmUrl, String sessionId) {
```

Comment:
Could you please check with Marko K if Flick recording is really needed to be merged?

Reply:
From his point of view it's not needed